cmd/env/setax.go

```diff
@@ -52,7 +52,7 @@ func init() {
 		"", "Member Name, example Service Account Name")
 	SetAxCmd.Flags().StringVarP(&memberType, "memberType", "m",
 		"serviceAccount", "memberType must be serviceAccount, user or group")
-	SetAxCmd.Flags().StringVarP(&role, "memberRole", "r",
+	SetAxCmd.Flags().StringVarP(&role, "memberRole", "",
```

Comment:
why was this removed? Is it because of the global `r` flag? If yes, then removing the existing `r` for `role` is breaking change right?